.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+- lts/*
+
+addons:
+  firefox: latest
+
+before_script:
+- export DISPLAY=:99.0
+- sh -e /etc/init.d/xvfb start
```

Comment:
Just saw in the docs - karma-firefox-launcher now supports a headless mode, so xvfb wouldn't be required. Maybe not worth changing now but fyi

Reply:
It must require some configuration because I forgot to put this in there originally. I think I would like to still run the tests in a really browser. Of course eventually we should use Sauce Labs or Browserstack for this.